modules/ROOT/pages/setup.adoc

```diff
@@ -0,0 +1,114 @@
+
+Information needed for each trading partner:
+
+* Partner name (usually the name of the company).
+
+* Identifiers (such as the AS2 Identifier, X12 ISA, and so on).
+
+* Message formats to send to one another (such as X12, and so on), and any associated validations with the need for acknowledgment (such as TA1, 997, 999, and so on).
+
+* Message types to use. For example, a purchase order in X12 is an 850 mesage. You also need to know which X12 version to use in your transactions (such as 004010, 005010, and so on).
+
+* The digital certificate (such as X.509).
+
+* Endpoints, depending on the direction of interactions with your partner.
+
+Information you'll need as the host organization:
+
+* Endpoints, depending on the direction of interactions with your partner.
+
+* Identifiers (such as the AS2 Identifier, X12 ISA, and so on). 
+
+* The digital certificate (such as X.509).
+
+* EDI message type preferences for your organization. [Example?]
+
+* Storage for your B2B message payloads. 
+
+== Set up Anypoint Partner Manager
+
+Use the information gathered in the checklist to associate runtimes, allocate user roles, and connect to your payload storage. 
+
+* Associate Mule Runtimes Manager for Deployment [link]
+
+* Allocate Partner Manager Roles [link]
+
+* Connect to Payload Storage [link]
+
+=== Associate Runtimes for Deployment
+
+Setup a server group via Anypoint Runtime Manager to deploy the B2B Gateway. The steps to setup your server group are in [Mule Runtime Manager documentation](https://docs.mulesoft.com/runtime-manager/server-group-create).
+
+**Note**
+
+The server group associated to your B2B Gateway must be named exactly **B2BServerGroup** in order for it to operate correctly. The name is case sensitive.
+
+=== Allocate Partner Manager Roles
+
+There are two Administrator roles for Anypoint Partner Manager. Both roles have unrestricted access, but one role works in a Production environment and one works in a Sandbox environment. 
+
+Partner Manager Admin (Production) - Can perform all operations across Partner Manager for all Production Environments.
+
+Partner Manager Admin (Sandbox) - Can perform all operations across Partner Manager for all Sandbox Environments.
+
+=== Connect to Payload Storage
+Anypoint Partner Manager enables you to connect your own storage (such as a database, Amazon S3, Azure Blob Storage, and so on) to store the incoming B2B message payload. You'll need to establish an API that provides Anypoint Partner Manager with a standardized way to connect to your unique storage.
+
+Download the pre-packaged API and integration implementation from Anypoint Exchange.[link from Anypoint Exchange for the pre-packaged API and integration app from Manik / Pablo]
```

Comment:
is line 101 explaining how to do what's stated in the 2nd sentence of line 99? Can we put the task in task format? (numbered list)?